ste/mgr-JobPartTransferMgr.go

```diff
@@ -107,6 +107,19 @@ type chunkFunc func(int)
 
 // jobPartTransferMgr represents the runtime information for a Job Part's transfer
 type jobPartTransferMgr struct {
+
+	// NumberOfChunksDone represents the number of chunks of a transfer
+	// which are either completed or failed.
+	// NumberOfChunksDone determines the final cancellation or completion of a transfer
+	atomicChunksDone uint32
+
+	// used defensively to protect against accidental double counting
+	atomicCompletionIndicator uint32
+
+	// how many bytes have been successfully transferred
+	// (hard to infer from atomicChunksDone because that counts both successes and failures)
+	atomicSuccessfulBytes uint64
```

Comment:
maybe try a windows 32 bit build to make sure the memory alignment works

Reply:
I _believe_ that, because I've moved all the atomics to the start of the struct, they are guaranteed to have correct alignment now (whereas before they were not).  So, I propose to close this comment, and leave testing for our already-scheduled 32-bit pre-release testing.